### DISCUSSION TITLE
unity version and on the unity store

### DISCUSSION THREAD

**Discussion by Brad3514** [visualdesigncafe/nature-renderer#17] [Help]
hi there, what version of nature renderer do u's support for 2018 LTS as we are using this unity for our VW for now, we have high hopes to migrate to 2020 unity but until then we are for now 2018, since we use the standard / built in renderer I like to purchase the right one for our needs from the unity store.

**Comment by maxplooi** [comment]
The official supported Unity version is currently Unity 2021 or later. Nature Renderer is still available for Unity 2019 and 2020, but official support is ending and it will be deprecated soon because Unity stopped supporting that version as well. We follow the same release/support schedule as the Unity LTS releases.
